artifact-management/commands/gem-test.yml

```diff
@@ -1,36 +1,16 @@
-description: Perform the actual gem build
+description: Perform simple gem test
 parameters:
-  bundler_version:
-    description: The Bundler version that should be used for "bundle install" and the actual test command
-    type: string
-    default: 1.17.3
-  gem_repo_host:
-    description: The host to which built packages should be pushed. 
-    type: env_var_name
-    default: AVVO_GEM_REPO_HOST
-  gem_repo_key:
-    description: The :rubygems_api_key used to access the gem repository host
-    type: env_var_name
-    default: AVVO_GEM_REPO_TOKEN
   gem_test_dir:
     description: The directory in which the gem tesets should run
     type: string
     default: .
-  packagecloud_read_token:
```

Comment:
I think that `setup-gem-environment` actually requires the `packagecloud_read_token`. By taking this away you aren't preventing me from specifying the `packagecloud_read_token` because (like you did with `bundler_version`) this setting is specified by environment variables, and the environment variable set in the CircleCI build context will still be visible in the executor process due to Avvo's `org-global` context configuration.

However, what you *have* done here is take away the _documentation_ that this environment variable is going to be consumed by the `gem-test` command, and that's not great. I had it listed here for that exact reason.